app/templates/base.html

```diff
@@ -13,8 +13,9 @@
   <ul class="navbar">
     {% block login %}
       <img src="ThriftyCarrotMockLogo.png" alt="Cartoon Carrot Logo">
-      <h1 style="color:#ffffff">Thrifty Carrot</h1>
+      <h1 style="color:#ffffff" >Thrifty Carrot</h1>
       {% if current_user.is_authenticated %}
+        <a href="{{ url_for('users.profile') }}" type="button" class="btn btn-login">User Profile</a>
```

Comment:
Maybe we should leave the login/logout button and instead if the user is logged in a bunny icon/avatar shows up that the can press to go to their profile